source/whats-new.txt

```diff
@@ -35,6 +36,26 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.4:
+
+What's New in 5.4
+-----------------
+
+New features of the 5.4 {+driver-short+} release include:
+
+.. important:: Deprecation Notice
+
+  - The ``collStats`` helper is deprecated. Use the :manual:`$collStats
```

Comment:
S: Looking at the [JIRA ticket](https://jira.mongodb.org/browse/NODE-4970), `collStats` is a helper command. Might be useful to specify here.

```suggestion
  - The ``collStats`` helper command is deprecated. Use the :manual:`$collStats
```